src/java.base/share/classes/java/net/URLClassLoader.java

```diff
@@ -89,12 +94,12 @@ public URLClassLoader(URL[] urls, ClassLoader parent) {
 
     /**
      * Constructs a new URLClassLoader for the specified URLs using the
-     * default delegation parent {@code ClassLoader}. The URLs will
-     * be searched in the order specified for classes and resources after
-     * first searching in the parent class loader. Any URL that ends with
-     * a '/' is assumed to refer to a directory. Otherwise, the URL is
-     * assumed to refer to a JAR file which will be downloaded and opened
-     * as needed.
+     * {@linkplain ClassLoader#getSystemClassLoader() default delegation
+     * parent class loader}. The URLs will be searched in the order
```

Comment:
As we are changing it then maybe we could change this to "using the system class loader as the parent". I think that would be much clearer than "default delegation parent ...". Same thing in SecureClassLoader.

Reply:
Done - updated these 2 places to use the "system class loader" reference.